src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,174 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String NAME_1 = "Adam";
+    private static final String NAME_2 = "Bob";
+    private static final String NAME_3 = "Kate";
+    private static final String CORRECT_PASSWORD_8_CHARS = "1234qwer";
+    private static final String CORRECT_PASSWORD_10_CHARS = "j48fb*&%mf";
+    private static final String CORRECT_PASSWORD_15_CHARS = "J&gf9Jg3IOj5La2";
+    private static final String INCORRECT_PASSWORD_1_CHAR = "g";
+    private static final String INCORRECT_PASSWORD_4_CHAR = "hfdl";
+    private static final String INCORRECT_PASSWORD_5_CHAR = "12345";
+    private static final int CORRECT_AGE_1 = 20;
+    private static final int CORRECT_AGE_2 = 25;
+    private static final int CORRECT_MINIMUM_AGE = 18;
+    private static final int INCORRECT_AGE_UNDER_MINIMUM = 16;
+    private static final int INCORRECT_AGE_ZERO = 0;
+    private static final int INCORRECT_AGE_LESS_ZERO = Integer.MIN_VALUE;
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_alreadyRegisteredUser_NotOk() {
+        User correctUser1 = getPreparedUser(
+                NAME_1, CORRECT_PASSWORD_8_CHARS, CORRECT_AGE_1);
+
+        String message = "When user already registered you must throw exception";
+        registrationService.register(correctUser1);

```

Comment:
You can not configure your storage before testing with the method you actually test.
Let's connect to the storage directly